File: Java/src/main/java/Exceptions/NoAvailableConstructorException.java
package Exceptions;

import java.lang.reflect.Type;

/**
 * Created by user on 22/9/2015.
 */
public class NoAvailableConstructorException extends RuntimeException {
    public NoAvailableConstructorException(Type contract) {
        super("The type that is trying to be resolved does not have a constructor available for injection: " + contract.getTypeName());
    }
}

File: C#/Container/Source/Exceptions/NoAvailableConstructorException.cs
using System;

namespace IoC.Source.Exceptions
{
    internal class NoAvailableConstructorException : Exception
    {
        public NoAvailableConstructorException(Type contract) : base($"The type that is trying to be resolved does not have a constructor available for injection: {contract.FullName}")
        {
        }
    }
}
